MidtransKit/code_usage_swift.md

```diff
@@ -45,6 +45,24 @@ import MidtransKit
 MidtransConfig.setClientKey(client_key, serverEnvironment: .sandbox OR .production, merchantURL: merchant_url)
 ```
 
+### Credit Card Payment Feature
+#### 2-Clicks
+```
+MidtransCreditCardConfig.setPaymentType(.twoclick, secure: true)
+MidtransCreditCardConfig.disableTokenStorage(true)
+```
+Parameter `secure` is for enabling 3D secure transaction, but for 2-clicks, actually it's forced to `true` even if you set it to `false`.
+You cannot use `tokenStorage` feature for 2-Click, so disable it.
+
```

Comment:
please also add link to merchant server implementation to save CC token https://github.com/veritrans/veritrans-android/wiki/Implementation-for-Merchant-Server